bench/workgen/workgen.cxx

```diff
@@ -774,53 +775,56 @@ int ThreadRunner::op_run(Operation *op) {
     }
     // Retry on rollback until success.
     while (retry_op) {
-	if (op->_transaction != NULL) {
+        if (op->_transaction != NULL) {
             if (_in_transaction)
                 THROW("nested transactions not supported");
             _session->begin_transaction(_session,
               op->_transaction->_begin_config.c_str());
             _in_transaction = true;
-	}
-	if (op->is_table_op()) {
-	    bool notfound_ok = false;
+        }
+        if (op->is_table_op()) {
             switch (op->_optype) {
             case Operation::OP_INSERT:
                 ret = cursor->insert(cursor);
                 break;
             case Operation::OP_REMOVE:
-		notfound_ok = true;
                 ret = cursor->remove(cursor);
+                if (ret == WT_NOTFOUND)
+                    ret = 0;
                 break;
             case Operation::OP_SEARCH:
                 ret = cursor->search(cursor);
+                if (ret == WT_NOTFOUND) {
+                    ret = 0;
+                    track = &_stats.not_found;
```

Comment:
I think the closing brace has a spacing error.

Reply:
Fixed.